presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for operations on timestamps in certain time zones (:issue:`11891`).
+* Fix correctness issue for :func:`array_intersect` and :func:`array_distinct` when input contains
+  both zeros and nulls.
+* Fix ``count(*)`` aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve the table scan performance for structured types.
+* Improve performance for :func:`array_intersect`.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function.
+* Add an optimizer rule to filter the window partitions before executing the window operators.
+* Remove ``ON`` keyword for :doc:`/sql/show-stats`.
+* Restrict ``WHERE`` clause in ``SHOW STATS`` to filters that can be pushed down to the connectors.
```

Comment:
Change SHOW STATS here to be a link so that it is consistent with the one above. Or change the one above to be text.